src/use-format-block.hook.js

```diff
@@ -6,10 +6,44 @@ export function useFormatBlock() {
   return {
     formatBlock(value) {
       if (value === document.queryCommandValue('formatBlock')) {
-        performCommandWithValue('div')
+        if (!!window.chrome && isInHTMLList()) {
+          removeHeader()
+        }
+        else {
+          performCommandWithValue('div')
+        }
       } else {
         performCommandWithValue(value)
       }
     },
   }
+}
+
+function isInHTMLList() {
+  let containerNode = window.getSelection().getRangeAt(0).commonAncestorContainer
+  let index = 0
+  let isList = false
+  while(isList == false && index < 2) {
+    if (containerNode.nodeType === Node.ELEMENT_NODE && containerNode.tagName === 'LI') {
+      return true
+    }
+    containerNode = containerNode.parentNode
+    index++
+  }
+}
+
+function removeHeader() {
+  let nodeToBeRemoved = window.getSelection().getRangeAt(0).commonAncestorContainer
+  let index = 0
+  while (nodeToBeRemoved.tagName !== 'H1' && index < 5) {
+    nodeToBeRemoved = nodeToBeRemoved.parentNode
+    index ++
+  }
+  if (index === 4) {
+    throw new Error(`The element does not appear to be wrapped by a header`)
```

Comment:
We wouldn't want to throw this error probably -- if it's not in a header, isn't is just okay to proceed with calling execCommand like normal?